tests/led/CMakeLists.txt

```diff
@@ -15,6 +15,6 @@ zephyr_include_directories(${CMAKE_SOURCE_DIR}/../..)
 project(led_test)
 
 target_sources(app PRIVATE 
-    main.c
+    bloodlight_based.c
```

Comment:
It isn't clear why you have renamed 'main.c' as 'bloodlight_based.c'. The intended meaning of the name 'bloodlight based' isn't very clear.

Reply:
I edited the commit description. See: https://github.com/CodethinkLabs/bloodlight-zephyr/pull/3/commits/8afe0b702ee2850ecb415b7cddd9758a6431e73f